src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
I don't think we should. Either we want the behavior then we have it in the core container tests or we decide to completely ditch it and then all code in the container adapters related to property injection needs to go. Everything in between smells like a half-baked approach